os/board/rtl8721csm/src/component/common/mbed/targets/hal/rtl8721d/serial_api.c

```diff
@@ -684,6 +684,24 @@ void serial_irq_set(serial_t *obj, SerialIrq irq, uint32_t enable)
 	}
 }
 
+/**
+  * @brief  Enable/Disable UART loopback mode.
+  * @param  obj: uart object define in application software.
+  * @param  enable/disable the loopback mode
+  *             @arg 0 disable
+  *             @arg 1 enable
+  * @retval none
+  */
+void serial_control_loopback(serial_t *obj, bool enable)

```

Comment:
I see you have implemented loopback for rtk driver. What about other boards?

Reply:
as i mentioned above
I didn't find the loopback support bit in artik and imxrt user manual, so don't know whether they support loopback or not so couldn't add this to these boards as of now.
i'll add this in mtk after merging of this PR.

Reply:
As Jeongchan said. `bool` requires stdbool.h.